### PR TITLE
add source code from hw4, rework data models, create python helper to…

### PR DESCRIPTION
**Pull request by dkapp7**
* This is my hw4 with the models updates to handle our sleep study
* reworkSubjectData.py was created to rework the shhs1-dataset-0.13.0.csv into a format similar to hw4 Data files
* TODO: unit tests are straight from hw4 (proly wont need)
* TODO: scala files are not refactored to the new model classes
* TODO: most of the data processing can happen with pandas. That way scala just needs to convert the data directly to rdd. Easier than working with sql statements.

### CODE REVIEW

**Comment by dkapp7** [comment]
@BobRiley1 This is the start of patient data processing.

File: subject-similarity-groups/reworkSubjectData.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# testVars = ['alcoh','cgpkyr','hi201a','hi201b','hi201c','hi201d','hi201e','hi216','htnderv_s1','htnderv_s2','parrptdiab','srhype','sa15','age_category_s1','age_category_s2','age_s1','age_s2','educat','ethnicity','gender','mstat','nsrrid','pptid','race','ace1','ace2','aced1','aced2','anar1a1','anar1b1','anar1c1','anar31','benzod1','benzod2','beta1','beta2','betad1','betad2','ccb1','diur2','diuret1','htnmed1','htnmed2','insuln1','insuln2','istrd1','istrd2','lipid1','lipid2','mlpd2','nsaid1','ntca1','ntg1','oaia2','ohga1','ohga2','ostrd1','ostrd2','tca1','tca2','bmi_s1','bmi_s2','height','hip','pm202','pm207','waist','weight','avg23bpd_s2','avg23bps_s2','diasbp','systbp','chol','hdl','trig','fev1','fvc','ai_all','avgsat','minsat','ahi_a0h3','ahi_a0h3a','ahi_a0h4','ahi_a0h4a','ahi_c0h3','ahi_c0h3a','ahi_c0h4','ahi_c0h4a','ahi_o0h3','ahi_o0h3a','ahi_o0h4','ahi_o0h4a','cai0p','oahi','oai0p','rdi0p','rdi3p','rdi3pa','rdi4p','rdinr3p','rdirem3p','overall_shhs2','nsupinep','pctlt75','pctlt80','pctlt85','pctlt90','slp_eff','slp_lat','slpprdp','slptime','supinep','times34p','timest1p','timest2p','waso']
```
Comment (macnb9) [inline subject-similarity-groups/reworkSubjectData.py]: Do we need anything from this file?

File: subject-similarity-groups/src/test/scala/edu/gatech/cse6250/graphconstruct/GraphLoaderTest.scala [hunk 2]
```diff
@@ -0,0 +1,234 @@
+package edu.gatech.cse6250.graphconstruct
```
Comment (macnb9) [inline subject-similarity-groups/src/test/scala/edu/gatech/cse6250/graphconstruct/GraphLoaderTest.scala]: We should probably just remove all of these test files

**Review by macnb9 — CHANGES_REQUESTED** [review]
Could you take out the data files (all the `csv` and `txt`) so that we don't have them living in the project? Then we can work on just pointing to whatever data we actually decide to input.

And where exactly are we going with this PR again? Since none of the models or data are anything we're going to be able to use right?

File: subject-similarity-groups/reworkSubjectData.py (hunk 1 above)
Comment (dkapp7) [reply subject-similarity-groups/reworkSubjectData.py]: yes - it generates the csv files from the sleep study raw data

File: subject-similarity-groups/src/test/scala/edu/gatech/cse6250/graphconstruct/GraphLoaderTest.scala (hunk 2 above)
Comment (dkapp7) [reply subject-similarity-groups/src/test/scala/edu/gatech/cse6250/graphconstruct/GraphLoaderTest.scala]: done

**Review by macnb9 — APPROVED** [review]

**Pull request merged**